### PR TITLE
Update licensing docs

### PR DESCRIPTION
**Pull request by pjlast**
Update the licensing docs to reflect the current state of Sourcegraph licensing, as well as what happens when a license expires after 5.3

### CODE REVIEW

File: docs/admin/licensing/index.mdx [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-License keys also need to be unique to a single instance of Sourcegraph. If the same license key is used across multiple instances, subsequent license checks will fail. If multiple license keys are required for dev/staging instances, contact customer support for additional license keys for these instances.
+#### Before Sourcegraph 5.3
 
-## How to update your license key
+Sourcegraph will revert to a free license, and any features that require an enterprise license will stop functioning. This could lead to data loss if some of these features were in use, so be sure to renew your license in advance!
 
-Any current Site Admin can update your license key by going to Site Admin -> [Site configuration](/admin/config/site_config)
+### How can we update our license key?
+
+Any current Site Admin can update your license key by going to Site Admin -> [Site configuration](../config/site_config.md)
```
Comment (MaedahBatool) [inline docs/admin/licensing/index.mdx:21]:
```suggestion
Any current Site Admin can update your license key by going to Site Admin -> [Site configuration](/admin/config/site_config)
```
The links will not work with `.md` extension, and I corrected the path. The parent folder, which is `admin` in this case, should be included. :)

**Review by MaedahBatool — CHANGES_REQUESTED** [review]
Left a very small link fix. Rest looks good. :)

**Comment by MaedahBatool** [comment]
Hey @pjlast left a very small link fix as a suggestion. Thanks for your contribution. :)

File: docs/admin/licensing/index.mdx (hunk 1 above)
Comment (pjlast) [reply docs/admin/licensing/index.mdx:21]: Ah gotcha, thanks!

**Review by MaedahBatool — APPROVED** [review]
LGTM 👍 Please merge when you are ready.

**Review by eseliger — APPROVED** [review]

**Pull request merged**